ColorPatterns/Scenes/StartMenu/StartMenuController.swift

```diff
@@ -24,6 +24,9 @@ final class StartMenuController: UIViewController {
     }
     
     @IBAction private func difficultyButtonPressed(_ sender: UIButton) {
+        // TODO: it's not a good idea to use tags to do such things
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/vandermesis/ColorPatterns/blob/6055d8cf2ff9610160e4bc6d457c28b6db6bd01c/ColorPatterns/Scenes/StartMenu/StartMenuController.swift#L27"></span>TODOs should be resolved (it’s not a good idea to use ta…).<br />
<code>todo</code> <code>StartMenuController.swift:27</code></td>
    </tr>
  </tbody>
</table>